.github/workflows/release.yml

```diff
@@ -0,0 +1,84 @@
+# This workflow is triggered by new releases and performes these steps: 
+# * it builds and uploads the CLI tool as an artifact to the release that triggered it
+# * it generates and deploys the documentation to Github pages and it builds and 
+#   publishes the sdk to Maven.
+# If the tag ends with "SNAPSHOT", only publishing to Maven takes place.
+
+name: Release
+
+on:
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+    - name: Cache Gradle packages
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.gradle/caches
+          ~/.gradle/wrapper
+        key: ${{ runner.os }}-gradle-${{ hashFiles('TransifexNativeSDK/**/*.gradle*', 'TransifexNativeSDK/**/gradle-wrapper.properties') }}
+        restore-keys: |
+          ${{ runner.os }}-gradle-
+    - name: Grant execute permission for gradlew
+      working-directory: ./TransifexNativeSDK
+      run: chmod +x gradlew
+    
+    - name: Assemble clitool jar
+      if:  ${{ !endsWith(github.event.release.tag_name, 'SNAPSHOT') }}
```

Comment:
We use lowercase in our tags, so let's use `snapshot` instead.

Reply:
The actual SDK version published to Maven though, should be "SNAPSHOT" because this is the norm. I think it's better that the actual tag is the same as the version. 

Either way, the `endsWith()` method is not case sensitive.